src/wt/main.js

```diff
@@ -1,3 +1,43 @@
+import { cpus } from "os"
+import { fileURLToPath } from "url"
+import { Worker } from "worker_threads"
+
+
 export const performCalculations = async () => {
-    // Write your code here
-};
\ No newline at end of file
+    const thisScriptPath = fileURLToPath(import.meta.url)
+    const workerScriptPath = `${thisScriptPath}/../worker`
+    
+    const workers = cpus()
+        .map(
+            (_, i) => new Promise((resolve, reject) => {
+                const worker = new Worker(workerScriptPath, {
+                    workerData: {
+                        n: 35 + i
```

Comment:
You should send incremental number starting from 10 to each worker. For example: on host machine with 4 cores you should create 4 workers and send 10 to first worker, 11 to second worker, 12 to third worker, 13 to fourth worker. 
10 not 35

Reply:
Yep, forgot to change it before commit:)